src/components/Organisation/Banner/index.tsx

```diff
@@ -73,20 +134,102 @@ const Banner: FC<IBanner> = ({ organisation }) => {
     return false;
   };
 
+  const handleOrgLogoChange = async (e: ChangeEvent<HTMLInputElement>) => {
+    e.preventDefault();
+    const { files } = e.target;
+    setFileState(files);
```

Comment:
I think this function can be generalised and we can pass the `setState` function also.